io.openems.edge.meter.api/src/io/openems/edge/meter/api/ElectricityMeter.java

```diff
@@ -153,21 +223,30 @@ public enum ChannelId implements io.openems.edge.common.channel.ChannelId {
 		 * Voltage L3.
 		 *
 		 * <ul>
-		 * <li>Interface: Meter Asymmetric
-		 * <li>Type: Integer
-		 * <li>Unit: mV
+		 * <li>Type: {@link OpenemsType#INTEGER}
+		 * <li>Unit: {@link Unit#MILLIVOLT}
 		 * </ul>
 		 */
 		VOLTAGE_L3(Doc.of(OpenemsType.INTEGER) //
 				.unit(Unit.MILLIVOLT) //
 				.persistencePriority(PersistencePriority.HIGH)), //
+		/**
+		 * Current.
```

Comment:
I am always unsure about the Current. What is it?
Is it max (l1,l2,l3) or is it mean(l1,l2,l3)? Right now  I do not see an added value by providing the CURRENT channel. 
Is it really needed? If so, may be we can have a more in depth textual explanation.

Reply:
Thanks @clehne. The problem applies to both `CURRENT` and `VOLTAGE`. I added these values initially, because the first devices where not even three-phased and these values are provided by the devices. Example: SOCOMEC e24 (https://www.socomec.com/files/live/sites/systemsite/files/SCP/6_gestion_energie/countis/countis_e/Countis_E28_48503055.html) registers 0xC588 and 0xC58C.

In fact it might be better to drop both `CURRENT` and `VOLTAGE` completely. Question is, if this should be done in this PR as well or in a separate one.

Reply:
Thanks for the explanation. As this PR already includes a lot of files I would suggest to handle the CURRENT and VOLTAGE drop in a separate PR.